index.html

```diff
@@ -2204,6 +2204,19 @@ <h2>Privacy</h2>
 		<section class="appendix" id="webidl">
 			<h2>WebIDL</h2>
 
+			<section id="webidl-intro" class="informative">
+				<h3>Introduction</h3>
+
+				<p>Although a Web Publication manifest is authored as [[json-ld]], user agents process this information
+					into an internal data structure representing the infoset in order to utilize the properties. The
+					exact manner in which this processing occurs, and how the data is used internally, is user
```

Comment:
Well, we might need to describe the processing itself if we keep the current section about processing a manifest to WebIDL.